tests/rptest/tests/metrics_test.py

```diff
@@ -54,6 +121,9 @@ def test_aggregate_metrics(self, aggregate_metrics):
         Verify that changing aggregate_metrics does preserve metric counts
 
         """
+        def filter_metrics(metrics):
+            return list(metric for metric in metrics
```

Comment:
info: this is the same as the list comprehension `[metric for metric in metrics if self.filter_metrics(metric)]`

Reply:
done